pkg/identity/cidr/identity.go

```diff
@@ -22,10 +22,11 @@ import (
 	"github.com/cilium/cilium/pkg/labels"
 	"github.com/cilium/cilium/pkg/labels/cidr"
 	"github.com/cilium/cilium/pkg/logging"
+	"github.com/cilium/cilium/pkg/logging/logfields"
 )
 
 var (
-	log = logging.DefaultLogger
+	log = logging.DefaultLogger.WithField(logfields.LogSubsys, "cidr-identity")
```

Comment:
`identity-cidr`? (If this is supposed to follow the path under `pkg/`, which seems to be the case for most of the other logs - might be easier/ more consistent)